model/internal/modeljson/error.go

```diff
@@ -116,9 +116,16 @@ func (e *Exception) marshalOne(w *fastjson.Writer, offset, parentOffset int) (in
 	if len(e.Stacktrace) != 0 {
 		maybeComma()
 		w.RawString(`"stacktrace":`)
-		if err := fastjson.Marshal(w, e.Stacktrace); err != nil {
-			return -1, err
+		w.RawByte('[')
+		for i, v := range e.Stacktrace {
+			if i != 0 {
+				w.RawByte(',')
+			}
```

Comment:
Is this right? If we're writing a JSON array, the format should be:

```javascript
[
{},
{}
]
```

With the logic above, a comma will be written after the first item has been written, which isn't right:

```javascript
[
{}
{},
]
```

Reply:
The comma is written before each element, except for before the first element.

Reply:
d'uh.... I clearly didn't read this right.